packages/envd/internal/logs/exporter/exporter.go

```
@@ -58,8 +60,17 @@ func printLog(logs []byte) {
 	fmt.Fprintf(os.Stdout, "%v", string(logs))
 }
 
-func (w *HTTPExporter) start() {
-	w.waitForMMDS(w.ctx)
+func (w *HTTPExporter) start(mmdsChan <-chan *host.MMDSOpts) {
+	mmdsOpts, ok := <-mmdsChan
+	if !ok {
+		fmt.Fprintf(os.Stderr, "channel closed when starting exporter")
+		return
+	}
+
+	if mmdsOpts == nil {
+		fmt.Fprintf(os.Stderr, "nil mmds opts received on channel when starting exporter")
```

this will silently fail in our case, we should have a retry logic here

---

this one is not resolved

---

forgot to remove that line